tests/test_auth.py

```diff
@@ -255,7 +256,7 @@ def test_jwt_decode_params(self, request_mock, jws_mock):
         request_mock.post.return_value = post_json_mock
         self.backend.authenticate(request=auth_request)
         calls = [
-            call('token', 'client_secret')
+            call(force_bytes('token'), force_bytes('client_secret'))
```

Comment:
Please change this for consistency to use ``smart_bytes`` like you do in the code.